src/main/java/com/darkona/adventurebackpack/util/TConstructTab.java

```diff
@@ -0,0 +1,132 @@
+package com.darkona.adventurebackpack.util;
+
+import java.util.List;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.entity.EntityClientPlayerMP;
+import net.minecraft.client.renderer.RenderHelper;
+import net.minecraft.client.renderer.entity.RenderItem;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+
+import org.lwjgl.opengl.GL11;
+import org.lwjgl.opengl.GL12;
+
+import com.darkona.adventurebackpack.init.ModNetwork;
+import com.darkona.adventurebackpack.network.GUIPacket;
+import com.darkona.adventurebackpack.reference.BackpackTypes;
+import com.darkona.adventurebackpack.reference.LoadedMods;
+
+import cpw.mods.fml.common.Optional;
+import tconstruct.client.tabs.AbstractTab;
+import tconstruct.client.tabs.TabRegistry;
+
+public class TConstructTab {
+
+    private static boolean enabled;
+
+    static {
+        enabled = LoadedMods.TCONSTRUCT;
+    }
```

Comment:
You already check if TiCon is loaded before calling methods of this class, so this feel redundant to me.